src/integrations/sdkinfo.ts

```diff
@@ -14,17 +15,30 @@ export class SdkInfo implements Integration {
    */
   public name: string = SdkInfo.id;
 
+  private _nativeSdkInfo: Package | null = null;
+
   /**
    * @inheritDoc
    */
   public setupOnce(addGlobalEventProcessor: (e: EventProcessor) => void): void {
-    addGlobalEventProcessor(event => {
+    addGlobalEventProcessor(async event => {
+      // The native SDK info package here is only used on iOS as `beforeSend` is not called on `captureEnvelope`.
+      // this._nativeSdkInfo should be defined a following time so this call won't always be awaited.
+      if (NATIVE.platform === 'ios' && this._nativeSdkInfo === null) {
+        try {
+          this._nativeSdkInfo = await NATIVE.fetchNativeSdkInfo();
```

Comment:
This value never changes at runtime so we could cache this and return the same value and avoid the bridge call

Reply:
Hmm I don't see any way of caching this other than modifying the JS at build time right? Storing it as a file outside JS would also need an async bridge call. This bridge call should only be called once so it should be fine IMO